resources/tags.yaml

```diff
@@ -0,0 +1,150 @@
+accounting:
+  title: 'Accounting'
+  description: 'Processing and communication of financial information'
+analytics:
+  title: 'Analytics'
+  description: 'Data or statistics to make intelligent marketing and business decisions'
+backend: 
+  title: 'Backend'
+  description: 'Backend'
+blogging: 
+  title: 'Blogging'
+  description: 'Informational site published with discrete entries'
+calendars:
+  title: 'Calendars'
+  description: 'Calendars'
+cloud_platform: 
+  title: 'Cloud Platform'
+  description: 'Offers hosting on the same supporting infrastructure'
+cms: 
+  title: 'CMS'
+  description: 'Content Management System'
+collaboration: 
+  title: 'Collaboration'
+  description: 'Collaboration'
+crm: 
+  title: 'CRM'
+  description: 'Customer Relationship Management'  
+customer_support:
+  title: 'Customer Support'
+  description: 'Range of customer services to assist users of a product'
+dashboard: 
+  title: 'Dashboard'
+  description: 'Control Panel'
+database:
+  title: 'Database'
+  description: 'Collection of schemas, tables, queries, reports, views and other objects'
+developer_tools: 
+  title: 'Developer Tools'
+  description: 'Developer Tools'
+digital_signature:
+  title: 'Digital Signature'
+  description: 'Scheme for demonstrating the authenticity of a digital message or documents'
+ecommerce:
+  title: 'eCommerce'
+  description: 'Facilitation of trading in products or services using computer networks'  
+education:
+  title: 'Education'
+  description: 'Education'
+email: 
+  title: 'Email'
+  description: 'Email'
+enterprise: 
+  title: 'Enterprise'
+  description: 'Enterprise'  
+erp:
+  title: 'ERP'
+  description: 'Enterprise resources planning'
+event_management:
+  title: 'Event Management'
+  description: 'Event Management'
+financial:
+  title: 'Financial'
+  description: 'Financial'
+forms: 
+  title: 'Forms'
+  description: 'Forms'
+hosting: 
+  title: 'Hosting'
+  description: 'Hosting'
+iot:
+  title: 'IoT'
+  description: 'Internet of Things'
+issue_tracking: 
+  title: 'Issue Tracking'
+  description: 'Issue Tracking'
+jobs:
```

Comment:
`jobs` I think it not the right name for the category where you place services like `oDesk`.
Since it's more like outsourcing.
But I don't have any good name at the moment.

I propose to delete it until we have such API, and decide then